dali/test/python/test_pipeline.py

```diff
@@ -902,37 +914,36 @@ def __init__(self, reader_type, batch_size, is_cached=False, is_cached_batch_cop
         elif reader_type == "readers.TFRecord":
             tfrecord = sorted(glob.glob(os.path.join(tfrecord_db_folder, '*[!i][!d][!x]')))
             tfrecord_idx = sorted(glob.glob(os.path.join(tfrecord_db_folder, '*idx')))
-            self.input = ops.readers.TFRecord(path=tfrecord,
-                                              index_path=tfrecord_idx,
-                                              shard_id=0,
-                                              num_shards=num_shards,
-                                              stick_to_shard=True,
-                                              skip_cached_images=skip_cached_images,
-                                              features={"image/encoded" : tfrec.FixedLenFeature((), tfrec.string, ""),
-                                                          "image/class/label": tfrec.FixedLenFeature([1], tfrec.int64,  -1)})
+            self.input = ops.readers.TFRecord(
+                path=tfrecord, index_path=tfrecord_idx, shard_id=0, num_shards=num_shards,
+                stick_to_shard=True, skip_cached_images=skip_cached_images, features={
+                    "image/encoded": tfrec.FixedLenFeature((), tfrec.string, ""),
+                    "image/class/label": tfrec.FixedLenFeature([1], tfrec.int64, -1)
+                })
         elif reader_type == "readers.Webdataset":
-            wds = [os.path.join(webdataset_db_folder, archive)
-                   for archive in ['devel-1.tar', 'devel-2.tar', 'devel-0.tar']]
+            wds = [
+                os.path.join(webdataset_db_folder, archive)
+                for archive in ['devel-1.tar', 'devel-2.tar', 'devel-0.tar']]
```

Comment:
What was wrong with the original formatting? The lines weren't too long or anything...

Reply:
Nothing really, reverted the change.